playground/infrastructure/helper.py

```diff
@@ -240,6 +241,7 @@ def _get_example(
   with open(filepath, encoding="utf-8") as parsed_file:
     content = parsed_file.read()
   content = content.replace(tag.tag_as_string, "")
+  link = Config.LINK_PREFIX + (filepath.replace(os.getenv("BEAM_ROOT_DIR"), "", 1))
```

Comment:
Please use string formating instead of concatenation

Reply:
Changed.